test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -10,6 +11,31 @@
 
 public class UtilsTest {
 
+    @Test
+    public void isAnyNull() throws Exception {
+        // empty list
+        assertNotNULL();
+
+        // only one object
+        assertAreNULL((Object) null);
+        assertNotNULL(1);
+        assertNotNULL("");
+        assertNotNULL("abc");
+
+        // all objects unique
```

Comment:
We are testing whether the objects passed into the `Utils.isAnyNull()` function contains a `NULL` object or not. Hence, the uniqueness of the objects does not matter in this case